### ISSUE TITLE
It seems dead

### ISSUE THREAD

**Issue by ValBaturin** [alexryndin/lis2#1]
Any updates so far? What do you think about _может использовать опшонал. передавать мутабельную ссылку. Можно еще воспользоваться паттерном интернал мутабилити или интериал мутабилити ну короче интерьерная мутабельность. Тогда тебе не нужно будет даже передавать мутабельную ссылку. Тебе достаточно будет передавать просто ссылку._

**Comment by ValBaturin** [comment]
anyway 22 days no updates

**Comment by ValBaturin** [comment]
can i be invited to the funeral?

**Comment by alexryndin** [comment]
No, the project is not dead. _использовать опшонал_ sounds great, we definitily need to implement that in a next release.

**Comment by ValBaturin** [comment]
i see this project is heavily moderated. Is it somehow related to the regime? What do you think about freedom of speech?
